openedx/core/djangoapps/theming/models.py

```diff
@@ -0,0 +1,39 @@
+"""
+Django models supporting the Comprehensive Theming subsystem
+"""
+from django.db import models
+from django.conf import settings
+from django.contrib.sites.models import Site
+
+
+class SiteTheme(models.Model):
```

Comment:
Sounds good. I just want to understand better the advantage that this separate model has over a field on the `SiteConfiguration` model both here and in Otto.